From: Michal Hocko <mhocko@suse.cz>
Subject: kabi fixup for mm: limit mmu_gather batching to fix soft lockups on !CONFIG_PREEMPT
Patch-mainline: never
References: bnc#791904

Hide a potential kabi change introduced by
patches.fixes/mm-limit-mmu_gather-batching-to-fix-soft-lockups-on-.patch.
mmu_gather is an internal data structure which is not used by any exported
function so it is safe to keep it hidden from kabi checker.

Signed-off-by:  Michal Hocko <mhocko@suse.cz>

---
 include/asm-generic/tlb.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/include/asm-generic/tlb.h
+++ b/include/asm-generic/tlb.h
@@ -102,7 +102,9 @@ struct mmu_gather {
 	struct mmu_gather_batch *active;
 	struct mmu_gather_batch	local;
 	struct page		*__pages[MMU_GATHER_BUNDLE];
+#ifndef __GENKSYMS__
 	unsigned int		batch_count;
+#endif
 };
 
 #define HAVE_GENERIC_MMU_GATHER
